### PR TITLE
feat: Upgrade Tensorboard from v1beta1 to v1

### PR DESCRIPTION
**Pull request by sasha-gitg**
- Added integration test for `aiplatform.Tensorboard`
- [b/206650730](b/206650730)

### CODE REVIEW

File: google/cloud/aiplatform/compat/types/__init__.py [hunk 1]
```diff
@@ -152,6 +158,13 @@
     prediction_service_v1,
     specialist_pool_v1,
     specialist_pool_service_v1,
+    tensorboard_v1,
+    tensorboard_service_v1,
+    tensorboard_data_v1,
+    tensorboard_experiment_v1,
+    tensorboard_run_v1,
+    tensorboard_service_v1,
```
Comment (yfang1) [inline google/cloud/aiplatform/compat/types/__init__.py:166]: dup'ed above

**Review by yfang1 — APPROVED** [review]

File: google/cloud/aiplatform/compat/types/__init__.py (hunk 1 above)
Comment (sasha-gitg) [reply google/cloud/aiplatform/compat/types/__init__.py:165]: done

File: tests/unit/aiplatform/test_custom_job.py [hunk 2]
```diff
@@ -260,24 +241,25 @@ def create_custom_job_mock_with_enable_web_access():
...
     with mock.patch.object(
-        job_service_client_v1beta1.JobServiceClient, "create_custom_job"
+        job_service_client.JobServiceClient, "create_custom_job"
     ) as create_custom_job_mock:
-        create_custom_job_mock.return_value = _get_custom_job_proto(
-            name=_TEST_CUSTOM_JOB_NAME,
-            state=gca_job_state_compat.JobState.JOB_STATE_PENDING,
-            version="v1beta1",
-        )
+        create_custom_job_mock.side_effect = RuntimeError("Mock fail")
```
Comment (morgandu) [inline tests/unit/aiplatform/test_custom_job.py:262]: What is this "Mock fail" side effect for?

**Review by morgandu — APPROVED** [review]
All changes look good, only have a question on Mock fail.

File: tests/unit/aiplatform/test_custom_job.py (hunk 2 above)
Comment (sasha-gitg) [reply tests/unit/aiplatform/test_custom_job.py:262]: This seems to be git diff surfacing the change in a confusing way.

`create_custom_job_mock_fail` hasn't changed, reference line 263 on the previous version. It's used to test that we handle failure from the service.

`create_custom_job_v1beta1_mock` was changed to `create_custom_job_mock_with_tensorboard` above.

**Pull request merged**